File: WS/artist.php
<?php
require_once("includes/class/Artist.class.php");		
if($_GET['action'] == "register"){
	$art = new Artist();
	$art->setArtista($_POST['artist']);
	$art->setVideo1($_POST['video1']);
	$art->setVideoName1($_POST['videoname1']);
	$art->setVideo2($_POST['video2']);
	$art->setVideoName2($_POST['videoname2']);
	$art->setVideo3($_POST['video3']);
	$art->setVideoName3($_POST['videoname3']);
	$art->setFestival($_POST['festival']);
	if($art->insertar())
		echo "OK";
	else
		echo "ERROR";
}

if($_GET['action'] == "getids"){
	$art = new Artist();
	$result = $art->getIds($_POST['festival']);
	while($row = mysql_fetch_array($result, MYSQL_NUM)){
		//0=msg, 1=fecha
		echo $row[0]."%%".utf8_decode($row[1])."</br>";
	}
}

if($_GET['action'] == "getvideos"){
	$art = new Artist();
	$art->getArtist($_POST['id']);
	echo $art->getVideo1()."++".$art->getVideoName1().",".$art->getVideo2()."++".$art->getVideoName2().",".$art->getVideo3()."++".$art->getVideoName2();
}
?>

File: Fex/src/com/fex/artist/Artist.java
package com.fex.artist;

import android.graphics.Bitmap;
import android.text.Html;

public class Artist {
	private String name;
	private int id;
	private String video1;
	private String video2;
	private String video3;
	private String videoName1;
	private String videoName2;
	private String videoName3;
	private String festivalId;
	private Bitmap img;
	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name=Html.fromHtml(name).toString();
	}
	public Bitmap getImg() {
		return img;
	}
	public void setImg(Bitmap img) {
		this.img = img;
	}
	public int getId() {
		return id;
	}
	public void setId(int id) {
		this.id = id;
	}
	public String getVideo1() {
		return video1;
	}
	public void setVideo1(String video1) {
		this.video1 = video1;
	}
	public String getVideo2() {
		return video2;
	}
	public void setVideo2(String video2) {
		this.video2 = video2;
	}
	public String getVideo3() {
		return video3;
	}
	public void setVideo3(String video3) {
		this.video3 = video3;
	}
	public String getFestivalId() {
		return festivalId;
	}
	public void setFestivalId(String festivalId) {
		this.festivalId = festivalId;
	}
	public String getVideoName1() {
		return videoName1;
	}
	public void setVideoName1(String videoName1) {
		this.videoName1 = Html.fromHtml(videoName1).toString();
	}
	public String getVideoName2() {
		return videoName2;
	}
	public void setVideoName2(String videoName2) {
		this.videoName2 = Html.fromHtml(videoName2).toString();
	}
	public String getVideoName3() {
		return videoName3;
	}
	public void setVideoName3(String videoName3) {
		this.videoName3 = Html.fromHtml(videoName3).toString();
	}
	
}
